torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -834,6 +834,8 @@ def summon_full_params(
             ``rank0_only=True``.
 
         Args:
+            module: Union[nn.Module, Tuple[nn.Module]]: Module or tuple
```

Comment:
We should use `Tuple[nn.Module, ...]` to denote variable-length tuple of homogeneous type `nn.Module`.